app/views/hub.nunjucks

```diff
@@ -17,7 +17,7 @@
       {% if gp.contact.telephone %}Telephone: {{ gp.contact.telephone }}<br />{% endif %}
       {% if gp.contact.email %}Email: {{ gp.contact.email }}<br />{% endif %}
       {% if gp.contact.fax %}Fax: {{ gp.contact.fax }}<br />{% endif %}
-      <a href="https://www.google.com/maps?q={{ gp.location.latitude }},{{ gp.location.longitude }}">See this surgery on a map</a>
+      <a href="https://www.google.com/maps/dir/{{ gp.name | safe | replace(" ", "+")}},{{ gp.address.addressLines | safe | replace(" ", "+") }},{{ gp.address.postcode | safe | replace(" ", "+")}}/@{{ gp.location.latitude }},{{ gp.location.longitude }},17z">See this surgery on a map</a>
```

Comment:
Sure, the principal is covered in this PR #47 and also in [connecting-to-services](https://github.com/nhsuk/connecting-to-services/blob/master/app/middleware/setLocals.js)

The PR adds a function to the view model but you'd be better off adding the link as it is done in `setLocals` (linked above).